docs/aaaCtl/API-Keys.md

```diff
@@ -0,0 +1,17 @@
+# API Keys
+
+**Organizational API Keys** 
+
+Organizational API Keys can be added to an Organizational account. From the Organization Dashboard tab, click the drop arrow on the right side of the API Keys field to display the Create API Key button. 
+   ![](img/orgapikey.png)
+
+Clicking on the Mange button allows Users to set read/write permissions and view the API key.
+   ![](img/manageapi.png)
```

Comment:
This file doesn't exist, it has been incorrectly named mangapi.png. Please change the filename.